slam-libraries/viam-orb-slam3/orbslam_server_v1_test.cc

```diff
@@ -322,29 +322,18 @@ BOOST_AUTO_TEST_CASE(ParseAndValidateArguments_config_invalid_use_live_data) {
     checkParseAndValidateArgumentsException(args, message);
 }
 
-// TODO: Remove no use_live_data test cases once integration tests have been
-// updated (See associated JIRA ticket:
-// https://viam.atlassian.net/browse/RSDK-1625)
-BOOST_AUTO_TEST_CASE(ParseAndValidateArguments_online_with_no_live_data) {
-    const vector<string> args{
-        "-data_dir=/path/to", "-config_param={mode=rgbd}",
-        "-port=20000",        "-sensors=color",
-        "-data_rate_ms=200",  "-delete_processed_data=false",
-        "-map_rate_sec=60"};
-    SLAMServiceImpl slamService;
-    utils::ParseAndValidateArguments(args, slamService);
-    BOOST_TEST(slamService.use_live_data == true);
-}
-
-BOOST_AUTO_TEST_CASE(ParseAndValidateArguments_offline_with_no_live_data) {
-    const vector<string> args{
-        "-data_dir=/path/to", "-config_param={mode=rgbd}",
-        "-port=20000",        "-sensors=",
-        "-data_rate_ms=200",  "-delete_processed_data=false",
-        "-map_rate_sec=60"};
-    SLAMServiceImpl slamService;
-    utils::ParseAndValidateArguments(args, slamService);
-    BOOST_TEST(slamService.use_live_data == false);
+BOOST_AUTO_TEST_CASE(ParseAndValidateArguments_config_no_use_live_data) {
```

Comment:
added one test for an empty `use_live_data` parameter. This should never be possible when run from RDK so can remove if desired.

Note: a similar test is not possible in carto as if a non-bool is sent it will immediately error before we perform our parsing (due to the the DEFINE_BOOL)

Reply:
I think if you already implemented it, leave it in :)